README.md

```diff
@@ -44,6 +44,8 @@ angular.module('myApp', ['angular-oauth2'])
   }]);
 ```
 
+ps: You can also configure service at runtime, in case you retrieve oauth server configuration from a ajax request.
```

Comment:
What do you think about using `OAuth` to avoid misinterpretation. So instead of:
**ps: You can also configure service at runtime...**
We could update it to:
**PS: You can also configure the `OAuth` service at runtime...**

And I would suggest to improve this a little, to show that this should be done on a `angular.run` block.